contrib-zypper/README.md

```diff
@@ -0,0 +1,35 @@
+# Zypper
+
+Defines [zypper][1] aliases.
+
+## Aliases
+
+* `zyp` # Zypper.
+* `zypc` # Cleans the cache.
+* `zypi` # Installs package(s).
+* `zypl` # Clean packages cache.
```

Comment:
Why `l`?

Reply:
could be renamed to zypcp or zypC ? which one is more clear?